.eslintrc.js

```diff
@@ -0,0 +1,117 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/eslint-recommended"
+    ],
+    "globals": {
```

Comment:
Did you have to add this section or did you copy it from the source? If the latter then can you see if it can be removed?